backend/api/Models/Concepts/AcquisitionFile/AcquisitionFilePersonMap.cs

```diff
@@ -11,15 +11,19 @@ public void Register(TypeAdapterConfig config)
                 .PreserveReference(true)
                 .Map(dest => dest.Id, src => src.AcquisitionFilePersonId)
                 .Map(dest => dest.IsDisabled, src => src.IsDisabled)
+                .Map(dest => dest.Person, src => src.Person)
+                .Map(dest => dest.PersonProfileType, src => src.AcqFlPersonProfileTypeCodeNavigation)
+                // TODO: Remove/refactor this flattened properties
                 .Map(dest => dest.PersonId, src => src.PersonId)
                 .Map(dest => dest.PersonProfileTypeCode, src => src.AcqFlPersonProfileTypeCode)
```

Comment:
For details view I used followed our convention of **not flattening** models. But I noticed that the create acquisition team implemented last sprint did use flattened models. Left a TODO here to refactor/change - FYI @Maheshbabu28 and @devinleighsmith

Reply:
Yes @Maheshbabu28 we should not be using flattened models.

Reply:
Sorry, what is flattened here though exactly? An acquisitionFilePerson has an isDisabled column and also a personId column

Reply:
The code table is flattened. Also while not strictly flat we don't expose foreign key ids eg personId in our models. That is to force the frontenis to use `model.person.I'd `

Reply:
Ah, I see. Yes, as long as we have both the personId and the person we are good here.

Reply:
Sounds good. Will remove the TODOs